app/javascript/widget/components/ChatHeaderExpanded.vue

```diff
@@ -1,6 +1,10 @@
 <template>
-  <header class="header-expanded" :style="{ background: widgetColor }">
+  <header class="header-expanded">
     <div>
+      <img
```

Comment:
Please remove this.

Reply:
Fixed, Along with min height issue for text area and branding text size  👍